### PR TITLE
Add 'mc admin update' command

### PR DESCRIPTION
**Pull request by harshavardhana**
Also this PR deprecates `service status` command

### CODE REVIEW

**Comment by harshavardhana** [comment]
This PR depends on https://github.com/minio/minio/pull/8070 - so this PR https://github.com/minio/minio/pull/8070 has to be merged first.

**Comment by poornas** [comment]
`  mc admin service - stop, restart or get status of MinIO server` should add `update`, admin docs also. It would be nice to correct PR desc above to `mc admin service update`

File: cmd/admin-service-update.go [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+FLAGS:
+  {{range .VisibleFlags}}{{.}}
+  {{end}}
+EXAMPLES:
+  1. Update MinIO server represented by its alias 'play'.
+     $ {{.HelpName}} play/
+`,
+}
+
+// serviceUpdateMessage is container for make bucket success and failure messages.
```
Comment (poornas) [inline cmd/admin-service-update.go]: ^^ comment needs to be fixed

File: cmd/admin-service-update.go [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+type serviceUpdateMessage struct {
+	Status         string `json:"status"`
+	ServerURL      string `json:"serverURL"`
+	CurrentVersion string `json:"currentVersion"`
+	UpdatedVersion string `json:"updatedVersion"`
+}
+
+// String colorized make bucket message.
```
Comment (poornas) [inline cmd/admin-service-update.go]: comment needs to be fixed

File: cmd/admin-service-update.go [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+		return console.Colorize("ServiceUpdate",
+			fmt.Sprintf("Server `%s` updated successfully from %s to %s",
+				s.ServerURL, s.CurrentVersion, s.UpdatedVersion))
+	}
+	return console.Colorize("ServiceUpdate",
+		fmt.Sprintf("Server `%s` already running the most recent version %s of MinIO",
+			s.ServerURL, s.CurrentVersion))
+}
+
+// JSON jsonified make bucket message.
```
Comment (poornas) [inline cmd/admin-service-update.go]: comment needs to be fixed

**Pull request merged**